# Existing code fails to build:
# libakumuli/query_processing/queryparser.cpp:82:53: error: invalid conversion from 'const char*' to 'std::basic_string<char>::size_type' {aka 'long unsigned int'} [-fpermissive]
# libakumuli/query_processing/queryparser.cpp:82:63: error: no matching function for call to 'std::basic_string<char>::basic_string(char*, const char*&)'
# Patch reverts to an earlier version from: https://github.com/akumuli/Akumuli/commit/ffab018fc5ce17395b20931cf9272c81f493aca6

--- libakumuli/query_processing/queryparser.cpp.orig	2020-05-19 18:47:08.000000000 +0545
+++ libakumuli/query_processing/queryparser.cpp	2022-08-06 09:25:58.000000000 +0545
@@ -79,7 +79,7 @@
     if (status != AKU_SUCCESS) {
         return status;
     }
-    series_.push_back(std::string(canonical.data(), keystr_end));
+    series_.push_back(canonical);
     return AKU_SUCCESS;
 }
 
